src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -169,148 +166,111 @@ Nethermind also has [detailed documentation](https://docs.nethermind.io), strong
+- Gets only the latest state while relying on trust in developers and consensus mechanism.
+- Client ready to use with current network state in a few minutes.
 
-#### Minimum requirements {#recommended-specifications}
+[More on Light clients](https://www.parity.io/blog/what-is-a-light-client/)
 
-- CPU with 2+ cores
-- 4 GB RAM minimum with an SSD, 8 GB+ if you have an HDD
-- 8 MBit/s bandwidth
+#### Snap sync
 
-#### Recommended specifications {#recommended-specifications}
+The latest approach to syncing the client, pioneered by Geth. Using dynamic snapshots served by peers retrieves all the account and storage data without downloading intermediate trie nodes and then reconstructs the Merkle trie locally.
 
-- Fast CPU with 4+ cores
-- 16 GB+ RAM
-- Fast SSD with at least 500 GB free space
-- 25+ MBit/s bandwidth
+- Fastest sync strategy, currently default in Geth and Nethermind 
+- Saves a lot of disk usage and network bandwidth without sacrificing security.
 
-The sync mode you choose will affect space requirements but we've estimated the disk space you'll need for each client below.
+[More on Snap](https://github.com/ethereum/devp2p/blob/master/caps/snap.md)
 
-| Client       | Disk size (fast sync) | Disk size (full archive) |
-| ------------ | --------------------- | ------------------------ |
-| Geth         | 400GB+                | 6TB+                     |
-| OpenEthereum | 280GB+                | 6TB+                     |
-| Nethermind   | 200GB+                | 5TB+                     |
-| Besu         | 750GB+                | 5TB+                     |
-| Erigon       | N/A                   | 1TB+                     |
 
-- Note: Erigon does not Fast Sync, but Full Pruning is possible (~500GB)
+#### Beam sync
 
-![A chart showing that GB needed for a full sync is trending up](./full-sync.png)
+Implemented by Nethermind and Trinity. Works like fast sync but also downloads the data needed to execute latest blocks, which allows you to query the chain within the first few minutes from starting.
 
-![A chart showing that GB needed for an archive sync is trending up](./archive-sync.png)
+- Syncs state first and enables you to query RPC in a few minutes.
+- Still in development and not fully reliable, background sync is slowed down and RPC responses might fail.
 
-These charts show how storage requirements are always changing. For the most up-to-date data for Geth and OpenEthereum, see the [full sync data](https://etherscan.io/chartsync/chaindefault) and [archive sync data](https://etherscan.io/chartsync/chainarchive).
+[More on Beam](https://medium.com/@jason.carver/intro-to-beam-sync-a0fd168be14a)
 
-### Ethereum on a single-board computer {#ethereum-on-a-single-board-computer}
+#### Optimistic sync
```

Comment:
This is a bit odd to mention here given it's the only CL sync strategy. Perhaps if we want to also mention CL sync we should more clearly break down the page into CL/EL sync modes and also include other CL sync modes?

Reply:
Agreed, I can break it down to two parts and should also add Checkpoint sync. Any other for CL?